src/utils/browser.js

```diff
@@ -12,12 +12,38 @@ export const getUrlParameter = name => {
   return decodeURI(results[1]) || 0;
 };
 
+export const getBrowserName = () => {
+  const userAgent = window.navigator.userAgent;
```

Comment:
I think it's better for us to use an external library like: https://www.npmjs.com/package/detect-browser and not do this on our own.
It will simplify things, and in case of something changes in the future - those libraries are often updated.
WDYT?

Reply:
Preffer not to add more dependencies to the project at the moment - we are just want to detect very specific browsers and that's doing the job. Libraries of that kind should be use to detect more complicated envs like device, os, browser version, and so on.